sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+# SIP-028: sBTC Bootstrap
+
+- **SIP Number:** 028
+- **Title:** A Decentralized and Programmable Asset Backed 1:1 with BTC 
+- **Consideration:** Technical, Economics
+- **Type:** Informational
+- **Status:** Draft
+- **Authors:**
+  - Andre Serrano
+  - Ashton Stephens
+  - Joey Yandle
+  - Mårten Blankfors
+  - Jesus Najera
+  - Jude Nelson
+  - Friedger Müffke
+  - Tycho Onnasch
+
+## Abstract
+sBTC is a SIP-010 token on the Stacks blockchain, backed 1:1 against BTC, and operated by a decentralized set of signers such that only a 70% majority can access the funds to maintain the protocol.
+
+While Bitcoin on its own cannot be used with smart contracts, sBTC provides a bridge of value from the Bitcoin Blockchain to the Stacks Blockchain to enable users who only own BTC to utilize the capability of smart contracts without switching to a new currency. This SIP aims to describe the high level sBTC system and the criteria for signer selection. 
+
+This first iteration of sBTC is purely at the application layer and does not affect network consensus or operation. This SIP does not attempt to describe the technical details of this or any subsequent sBTC releases; it only seeks to get community approval for the requirements of the sBTC bootstrap signers.
```

Comment:
The abstract still reads like the authors can't decide of this SIP is going to be a technical SIP describing a particular contract, or a SIP that only focuses on soliciting signers for a forthcoming system.

It sounds like the latter is the goal, and not the former?  If so, then I think this abstract needs to be rewritten.

> ## Abstract

> This SIP takes the position that Stacks can play a key role in offering a rich programming environment for Bitcoin with low-latency transactions.  This would be achieved with a new wrapped Bitcoin asset, called sBTC, which would be implemented on Stacks 3.0 and later as a SIP-010 token.  Stacks today offers a smart contract runtime for Stacks-hosted assets, and the forthcoming Stacks 3.0 release [link to SIP-021] provides lower transaction latency than Bitcoin for Stacks transactions.  By providing a robust BTC-wrapping mechanism based on threshold signatures, users would be able to lock their real BTC on the Bitcoin chain, instantiate an equal amount of sBTC tokens on Stacks, use these sBTC tokens on Stacks, and eventually redeem them for real BTC at 1:1 parity, minus the cost of the relevant blockchain transaction fees.

> This is the first of several SIPs that describe such a system.  This SIP describes the threshold signature mechanism, and solicits from the ecosystem both a list of signers and the criteria for vetting them.  These sBTC signers would be responsible for collectively holding all locked BTC, and redeeming sBTC for BTC upon request.  Given the high-stakes nature of their work, the authors of this SIP believe that such a wrapped asset can only be made to work in practice if the Stacks ecosystem members can reach broad consensus on how these signers are chosen.  Thus, the first sBTC SIP put forth for activation concerns the selection of sBTC signers.

> This SIP outlines but does not describe in technical detail the workings of the first sBTC system.  A separate SIP will be written to do so if this SIP successfully activates.